src/main/java/uk/gov/hmcts/reform/coh/domain/Answer.java

```diff
@@ -12,16 +15,44 @@
     @GeneratedValue(strategy=GenerationType.SEQUENCE, generator="seq_answer_id")
     @Id
     @Column(name = "answer_id")
+    @JsonProperty("answer_id")
     private Long answerId;
 
     @Column(name = "answer_text")
+    @JsonProperty("answer_text")
     private String answerText;
 
     @ManyToOne(fetch=FetchType.LAZY)
     @JoinColumn(name = "question_id")
     @JsonIgnore
     private Question question;
 
+    @OneToOne(fetch = FetchType.EAGER)
+    @JoinColumn(name = "answer_state_id")
+    @JsonProperty("current_answer_state")
+    private AnswerState answerState;
+
+    @OneToMany(mappedBy = "answer",
+            cascade = CascadeType.ALL,
+            orphanRemoval = true)
+    @JsonIgnore
+    private List<AnswerStateHistory> answerStateHistories = new ArrayList<>();
+
+
+    public void addState(AnswerState answerState){
+        this.answerState = answerState;
```

Comment:
Don't think we should rely on the addState to set the state of the Answer. That's the risk of mixing independent actions together. I think it would be better to separately set the state and adding an event